web/package.json

```diff
@@ -24,23 +23,28 @@
     "gulp-util": "^3.0.4",
     "json-loader": "^0.5.4",
     "lodash": "^4.0.0",
+    "pure-render-decorator": "^1.1.1",
```

Comment:
Are there any plans to use React.PureComponent? That would require v15.3.0 though.